src/QMCHamiltonians/NonLocalECPotential.deriv.cpp

```diff
@@ -85,7 +85,7 @@ NonLocalECPComponent::RealType NonLocalECPComponent::evaluateValueAndDerivatives
     std::fill(dlogpsi_vp.begin(), dlogpsi_vp.end(), 0.0);
     psi.evaluateDerivativesWF(W, optvars, dlogpsi_vp);
     for (int v = 0; v < dlogpsi_vp.size(); ++v)
-      dratio(v, j) = dlogpsi_vp[v];
+      dratio(v, j) = (dlogpsi_vp[v] - dlogpsi[v]);
```

Comment:
Please explain this change

Reply:
I will connect https://github.com/QMCPACK/qmcpack/blob/245b753f3096b66724e3cd84770c53f690730c85/src/QMCWaveFunctions/WaveFunctionComponent.h#L580

See the comment line above the function. The subtraction should be inside the function call.
This subtraction is moved from below and then I can make a local branch calling batched and non-batched.